base/applications/mscutils/eventvwr/lang/ja-JP.rc

```diff
@@ -123,13 +123,13 @@ BEGIN
     EDITTEXT IDC_LOGNAME, 67, 23, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
     LTEXT "ログファイル(&F):", IDC_STATIC, 7, 41, 60, 8
     EDITTEXT IDC_LOGFILE, 67, 39, 178, 12, ES_LEFT | ES_AUTOHSCROLL | ES_READONLY | WS_CHILD | WS_VISIBLE | WS_BORDER | WS_TABSTOP
-    LTEXT "サイズ(&S):", IDC_STATIC, 7, 57, 60, 8
+    LTEXT "サイズ:", IDC_STATIC, 7, 57, 60, 8
     EDITTEXT IDC_SIZE_LABEL, 67, 57, 178, 12, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
-    LTEXT "作成日時(&C):", IDC_STATIC, 7, 69, 60, 8
```

Comment:
Why deleting access keys?

Reply:
We are making better Windows.
Please don't be a copy cat.

Reply:
1. They aren't present in english translations;
2. They aren't there on windows:
![image](https://github.com/reactos/reactos/assets/1969829/1b820f5a-bf91-4389-91d8-9442ec2a44b8)
probably because there :point_up: the corresponding fields (date etc.) are not readonly but selectable edittexts, but static labels.
3. However I made that eventvwr dialog in such a way that these fields would be selectable (which is an improvement over the win2k3 eventvwr dialog). And on Vista+, the equivalent dialog has such selectable readonly edittexts:
![image](https://github.com/reactos/reactos/assets/1969829/8a8f0ddd-82c9-407f-a743-42d93d48cab9)
But here indeed, we observe that there exists accelerators for these.

Reply:
Yes, Hermes my reason was exactly the 1.) and 2.) that you pointed out.

In case we would add any accels here in the future for making it selectable, then we should do that (together at the same moment) for all languages. Until then I do think our life is easier without them.

Reply:
@katahiromz I will act as described. I will delete them in this PR, merge the PR, and will create a follow-up-PR today that restores your &S&C&M&A, and do it for all languages then. We will keep it with the accels (and for some langs with the comments for now) in master, I will leave that part out for the backports.

Reply:
see https://github.com/reactos/reactos/pull/7111 for continued discussion.